packages/rainbowkit/src/wallets/walletConnectors/portalWallet/portalWallet.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+export interface BraveWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const portalWallet = ({
+  chains,
+  shimDisconnect,
+}: BraveWalletOptions): Wallet => ({
+  id: 'portal',
+  name: 'Ripio Portal',
+  iconUrl: async () => (await import('./portal.svg')).default,
+  iconAccent: '#65ffb5',
+  iconBackground: '#fff',
+  installed:
+    typeof window !== 'undefined' && window.ethereum?.isPortal === true,
+  downloadUrls: {
+    browserExtension:
+      'https://chrome.google.com/webstore/detail/ripio-portal/ddamhapapianibkkkcclabgicmpnpdnj',
+    qrCode: 'https://www.ripio.com/ar/portal/',
+  },
+  createConnector: () => ({
+    connector: new InjectedConnector({
```

Comment:
Expecting WalletConnect connector support for Ripio mobile here as well. Reference MetaMask connector for a multi-platform connector implementation

Reply:
Currently our Ripio Portal wallet is only an extension (it is independent of the Ripio mobile app). I removed the QR code and only left the URL of the extension to avoid confusion.